emails/models.py

```diff
@@ -865,14 +867,11 @@ def make_domain_address(
             # Only check for bad words if randomly generated
         assert isinstance(address, str)
 
+        # update first_emailed_at indicating alias generation impromptu.
```

Comment:
I'm not sure if the comment is needed, because the code is descriptive, but it sounds wrong when read aloud. Maybe?
```suggestion
        # Set first_emailed_at when mask is first seen in an email, instead of created before use.
```

Reply:
Will opt to remove the comment since the refactor makes it more clear on when `first_emailed_at` should be used.

Reply:
Completed on cd1fd9ef06ac19b2bbe01d4bf2b59dc721cc561c